src/sidebar/components/test/sidebar-content-test.js

```diff
@@ -265,8 +357,8 @@ describe('sidebar.components.sidebar-content', function() {
         $scope.$digest();
       });
 
-      it('selectedAnnotationCount is > 0', function() {
-        assert.equal(ctrl.selectedAnnotationCount(), 1);
+      it('annotationsAreSelected is true', function() {
```

Comment:
Didn't convert this to `() =>` here cause I figured I'd save that for the refactor of these tests and all I did was update the description slightly.